src/html/util.rs

```diff
@@ -232,8 +232,12 @@ impl NamespacedGlobalSymbols {
 pub enum UrlResolveKind<'a> {
   Root,
   AllSymbols,
-  Category(&'a str),
-  File(&'a ShortPath),
+  Category {
+    category: &'a str,
+  },
+  File {
+    file: &'a ShortPath,
+  },
```

Comment:
this is done because serde cant handle tuple-style enum variants when using the `tag` attribute